lib/src/configuration/better_player_data_source.dart

```diff
@@ -35,6 +35,8 @@ class BetterPlayerDataSource {
   ///Should player use hls /das audio tracks
   final bool? useAsmsAudioTracks;
 
+  final bool? useInitialMaxBitrate;
```

Comment:
potrzebujemy zapakowac to w jakas klase dla dalszej mozliwosci rozszerzania tego featura